apps/application-system/form/src/routes/Application.tsx

```diff
@@ -21,19 +29,47 @@ export const Application = () => {
     skip: !id,
   })
 
+  const application = data?.getApplication
+  const nationalRegistryId = userInfo?.profile?.natreg
+
+  useEffect(() => {
```

Comment:
I feel like this belongs in the application form lib, not in the route. Maybe we should move this logic and the application query down one level, so the Route only needs to add 
`
<ApplicationForm
applicationId={applicationId}
nationalRegistryId={nationalRegistryId} />
`